posts/templates/posts/_post.html

```diff
@@ -41,11 +41,30 @@
             {% endif %}
             <span class="pull-right">
                 {# author actions (edit, delete, ...) #}
+            {% if request.user.pk == user.pk %}
+                <a id="delete-button" href="{% url 'posts:delete' pk=post.original_or_self.pk %}"  role="button" ><span class="glyphicon glyphicon-trash"></span></a>
```

Comment:
What if I want to delete my repost? Should be just `pk=post.pk`, instead of `pk=post.original_or_self.pk`.